modules/graphql_core/src/Plugin/GraphQL/Fields/EntityQuery/EntityQueryCount.php

```diff
@@ -27,7 +27,7 @@ public function resolveValues($value, array $args, ResolveContext $context, Reso
     if ($value instanceof QueryInterface) {
       // Clone the query and execute it as a count query.
       $clone = clone $value;
-      yield (int) $clone->range()->count()->execute();
+      yield (int) $clone->accessCheck(FALSE)->range()->count()->execute();
```

Comment:
This is a potential security issue, the default of not calling `accessCheck` is to apply access handling (i.e. `accessCheck(TRUE)`)

Reply:
Agreed, this looks a bit dangerous. Can we use `accessCheck(TRUE)`? Probably PHPStan or some tool indicated that an access check should be added here?

Reply:
I have already changed it. Thanks for the suggestion. Phpstam suggested me to change this.